### PR TITLE
feature/static-folder-for-frontend

### PR DESCRIPTION
**Pull request by dabeeeenster**

### CODE REVIEW

**Review by kyle-ssg — CHANGES_REQUESTED** [review]
Needs plugins.js/webpack.config.django.prod.js to be adjusted, needs testing with npm run bundle and npm start

**Comment by kyle-ssg** [comment]
> Needs plugins.js/webpack.config.django.prod.js to be adjusted, needs testing with npm run bundle and npm start

image

**Comment by dabeeeenster** [comment]
> Needs plugins.js/webpack.config.django.prod.js to be adjusted, needs testing with npm run bundle and npm start

This is fixed - tested and working here - also the django AIO image is working now too.

File: frontend/web/components/LegalAside.js [hunk 1]
```diff
@@ -18,15 +18,15 @@ const LegalAside = class extends Component {
...
-							Bullet Train
+                    <img height={34} src="/static/images/icon-2.png" alt="Flagsmith Logo"/>
+							Flagsmith
 
                 </Row>
             </Link>
         </div>
         <Flex>
             <Flex className="links">
-                <h3 className="link-header">Agreeements</h3>
```
Comment (matthewelwell) [inline frontend/web/components/LegalAside.js:29]: Lol

**Review by matthewelwell — APPROVED** [review]
Looks good to me.

File: frontend/web/components/LegalAside.js (hunk 1 above)
Comment (dabeeeenster) [reply frontend/web/components/LegalAside.js:29]: The quality was immense in 2017

**Review by kyle-ssg — APPROVED** [review]

**Pull request merged**